### ISSUE TITLE
After creating new object or clicking 'c', global redirect shortcuts don't work

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#6631]
**Scenario 1:**
1. Log in
2. Create new object
3. Try to go to other page using shortcuts listed in Search sub-tab

**Scenario 2:**
1. Log in
2. Click `c` on keyboard (new row at the top of the table appears like user would like to create new object)
3. Try to go to other page using shortcuts listed in Search sub-tab
**Actual:** No action, seems like cursor is still focused on table or row in table

**Expected:** User should be redirected to tab accordingly to pressed key combinations listed in Search

**Comment by Nabhag8848** [comment]
can't reproduce. Looks like already fixed, can be closed 

https://github.com/user-attachments/assets/f3ce8cdb-327d-4c8c-91b9-af3131fb096c

**Comment by BOHEUS** [comment]
No, it's not fixed, I can still reproduce it, also, I didn't mention anything about going to Settings and creating a entirely new object. Not only that, you tried to reproduce only first scenario where I wrote 2 scenarios (maybe there are more ways to reproduce the bug but I didn't find more).

Here's proof the bug is still reproducible (even on demo)
- scenario 1
[Screencast from 2024-08-25 15-36-41.webm](https://github.com/user-attachments/assets/af0537b6-37fe-4923-b274-a9a93cbede06)
- scenario 2
[Screencast from 2024-08-25 15-38-31.webm](https://github.com/user-attachments/assets/f6ff8841-2a2e-40b7-8518-8e658d85d641)

**Comment by Nabhag8848** [comment]
Thanks, you meant Create new object-record. **Create new object** was confusing to me.

**Comment by Bonapara** [comment]
@lucasbordeau linked to shortcuts

**Comment by BOHEUS** [comment]
@lucasbordeau done

**Comment by lucasbordeau** [comment]
Thanks for the update.

I tried to reproduce by following your videos, sadly I couldn't reproduce.

Could you try finding the precise sequence of actions that creates this shortcut freeze ? 

It could help to set `DEBUG_HOTKEY_SCOPE` to true in `useScopedHotkeyCallback` so you have logs that indicates what is the current hotkey scope when you press some keys.

**Comment by BOHEUS** [comment]
@lucasbordeau that's what I get when `DEBUG_HOTKEY_SCOPE` is set to `true`
![image](https://github.com/user-attachments/assets/d6da4338-471a-41ac-a335-2286b21cb48b)
For comparison, that's what I get when I can use shortcuts to jump between different tabs (messages in green)
![image](https://github.com/user-attachments/assets/9257ec3a-086e-43b8-a339-2d092eb34d0e)

**Comment by lucasbordeau** [comment]
Are you sure that you have no soft focus left on any cell when trying to input those keys ? 

Otherwise this would mean that even if we exit soft focus mode, the hotkey scope is still on soft focus mode, which would need to be fixed.

**Comment by BOHEUS** [comment]
I don't think I have soft focus on any cell after creating the record but how can I check it to make sure my statement is true?

**Comment by FelixMalfait** [comment]
Closing since there were quite a lot of changes since then and the behavior is now different (new record opens in side panel)
